### DISCUSSION TITLE
How can I start my own instance on my machine

### DISCUSSION THREAD

**Discussion by Av32000** [anuraghazra/github-readme-stats#2198] [Q&A] [answered]
Hi, I'm trying to figure out how to run my own instance of the project on my machine. I have cloned the repo on my machine but I can't find how to run the program. I would like not to go through Vercel. Is this possible and if so how? Thank you in advance.

**Comment by rickstaa** [comment, marked as answer]
@Av32000 Thanks for your question. To run the project locally on your own machine, use the `npm i` and `vercel dev` commands (see https://github.com/anuraghazra/github-readme-stats/blob/master/CONTRIBUTING.md#local-development). If you want to run it in a docker container, please check https://github.com/anuraghazra/github-readme-stats/discussions/1975. If you want to run it as an action, please check #2179.

**Reply by Av32000** [reply]
Thank You !
